README.md

```diff
@@ -1,4 +1,6 @@
-[![Build status](https://badge.buildkite.com/b0041826d71f5484c22145f44b3eac12357f51feb6ba6abb57.svg?branch=master)](https://buildkite.com/bazel/bazel-integration-testing-postsubmit)
+Bazel CI | Travis CI
```

Comment:
Wdyt about adding a sentence saying we expect people to run the lint fix before submitting PRs?

Reply:
Ok, but I put it in CONTRIBUTING.md, following https://github.com/bazelbuild/rules_docker/blob/master/CONTRIBUTING.md

Reply:
sounds good